packages/create-astro/src/actions/help.ts

```diff
@@ -11,12 +11,12 @@ export function help() {
 				['--template <name>', 'Specify your template.'],
 				['--install / --no-install', 'Install dependencies (or not).'],
 				['--git / --no-git', 'Initialize git repo (or not).'],
-				['--yes (-y)', 'Skip all prompt by accepting defaults.'],
-				['--no (-n)', 'Skip all prompt by declining defaults.'],
+				['--yes (-y)', 'Skip all prompts by accepting defaults.'],
+				['--no (-n)', 'Skip all prompts by declining defaults.'],
 				['--dry-run', 'Walk through steps without executing.'],
 				['--skip-houston', 'Skip Houston animation.'],
-				['--ref', 'Choose astro branch (default: latest).'],
-				['--fancy', 'Enable full unicode support for Windows.'],
+				['--ref', 'Choose astro Astro branch (default: latest).'],
```

Comment:
```suggestion
				['--ref', 'Choose astro branch (default: latest).'],
```
 duplicated word. (And, we would typically use lowercase to refer to the repository, so I think this is the one we'd keep.)